### DISCUSSION TITLE
;dtmfmode=auto is not documented in Wiki/Accounts

### DISCUSSION THREAD

**Discussion by juha-h** [baresip/baresip#2980] [Q&A] [answered]
`;dtmfmode=auto` is not documented in Wiki/Accounts https://github.com/baresip/baresip/wiki/Accounts.  What does it mean?

**Comment by cspiel1** [comment, marked as answer]
Updated https://github.com/baresip/baresip/wiki/Accounts#dtmfmode

**Comment by juha-h** [comment]
Thanks, I added `auto` value to two missing places.
